Makefile

```diff
@@ -31,6 +31,7 @@ COMMIT_NO_SHORT := $(shell git rev-parse --short HEAD 2> /dev/null || true)
 COMMIT := $(if $(shell git status --porcelain --untracked-files=no),${COMMIT_NO}-dirty,${COMMIT_NO})
 VERSION_COMMIT := $(if $(COMMIT),$(VERSION)-$(COMMIT),$(VERSION))
 ARCH := $(shell go env GOARCH)
+BUILDTAGS := seccomp
```

Comment:
It might be useful to build the agent without seccomp (but make it the default) so maybe you could add support for `WITH_SECCOMP=no` or something.

wdyt @bergwolf, @sboeuf?

It might also be good to add the fact that the agent has been build with seccomp support to the announce fields that are logged on agent startup:

- https://github.com/kata-containers/agent/blob/master/agent.go#L610

Reply:
Don't get me wrong - I'd like it enabled :)

But some folk may not want it and be happy to use a system like the current one without having to hack the makefiles and osbuilder scripts. It would help to know the performance and memory density impact of using seccomp.

/cc @grahamwhaley.

Reply:
Indeed, I think we should at least assess if there is any density (footprint) or boot speed impact.
You could try using https://github.com/kata-containers/tests/tree/master/metrics/report to do a before/after comparison to see if there is any measurable (and repeatable) difference.
If there is, then maybe we will discuss having two agent versions.
If there is not, we would probably just go ahead and enable it.

(traditionally we have just moved ahead and enabled things, but that does mean over time we suffer 'death of a thousand cuts' and our footprint and boot speed slowly creeps up and up.... we should make these conscious decisions ;-) )